src/lib/css/main.css

```diff
@@ -0,0 +1,8 @@
+body {
+	margin: 0;
+}
+
+:root {
+	--very-dark-pink: #540028;
+	--dark-pink: #a1004d;
```

Comment:
Instead of rgb literals consider the hsl color spaces which would make it clearer how the two colors are related:

```css
:root {
  --very-dark-pink: hsl(331.4, 100%, 16.5%);
  --dark-pink: hsl(331.3, 100%, 31.6%);
}
```

Hue could even be altered to match here.